htdocs/fourn/class/fournisseur.facture-rec.class.php

```diff
@@ -0,0 +1,2209 @@
+            $sql .= ', ' .(!empty($this->fk_project) ? $this->fk_project : 'NULL');                                                 // Fields declarded on creation
+            $sql .= ', ' .(!empty($facfourn_src->fk_account) ? $facfourn_src->fk_account : 'NULL');
+            $sql .= ', ' .($this->cond_reglement_id > 0 ? (int) $this->cond_reglement_id : 'NULL');
+            $sql .= ', ' .($this->mode_reglement_id > 0 ? (int) $this->mode_reglement_id : 'NULL');
+            $sql .= ", '".($facfourn_src->date_echeance > 0 ? $this->db->idate($facfourn_src->date_echeance) : 'NULL')."'";         // date_lim_reglement
+            $sql .= ', ' .(!empty($this->note_private) ? "'".$this->db->escape($this->note_private)."'" : 'NULL');                  // Fields declarded on creation
+            $sql .= ', ' .(!empty($this->note_public) ? "'".$this->db->escape($this->note_public)."'" : 'NULL');                    // Fields declarded on creation
+            $sql .= ', ' .(!empty($this->model_pdf) ? "'".$this->db->escape($this->model_pdf)."'" : 'NULL');                        // Fields declarded on creation
+            $sql .= ', ' . $facfourn_src->fk_multicurrency;
+            $sql .= ", '".$this->db->escape($facfourn_src->multicurrency_code)."'";
+            $sql .= ', ' . (float) $facfourn_src->multicurrency_tx;
+            $sql .= ', ' . $this->usenewprice;                                                                                      // Fields declarded on creation
+            $sql .= ', ' . (int) $this->frequency;                                                                                  // Fields declarded on creation
+            $sql .= ", '".$this->db->escape($this->unit_frequency)."'";                                                             // Fields declarded on creation
+            $sql .= ', ' .(!empty($this->date_when) ? "'".$this->db->idate($this->date_when)."'" : 'NULL');                         // Fields declarded on creation
+            $sql .= ', ' .(!empty($this->date_last_gen) ? "'".$this->db->idate($this->date_last_gen)."'" : 'NULL');                 // Fields declarded on creation
+            $sql .= ', ' . $this->nb_gen_done;                                                                                      // Fields declarded on creation
+            $sql .= ', ' . $this->nb_gen_max;                                                                                       // Fields declarded on creation
+            $sql .= ', ' . $this->auto_validate;                                                                                    // Fields declarded on creation
+            $sql .= ', ' . $this->generate_pdf;                                                                                     // Fields declarded on creation
+            $sql .= ')';
+
+            if ($this->db->query($sql)) {
+                $this->id = $this->db->last_insert_id(MAIN_DB_PREFIX. 'facture_fourn_rec');
```

Comment:
C'est dommage, t'aurais pu utiliser le $this->table_element

Reply:
Pour la mention "facture_fourn_rec" tu veux dire ?

Reply:
Done